cigaradvisor/blocks/carousel/carousel.css

```diff
@@ -1,4 +1,4 @@
-.carousel-wrapper {
+main .section .carousel-wrapper {
```

Comment:
Why do you need to override the wrapper?

Reply:
![image](https://github.com/hlxsites/famous-smoke-cigaradvisor/assets/53608019/429bfc05-c686-437d-95ef-0630a57ab81a)